### DISCUSSION TITLE
Intended way to obtain matched route/firewall names?

### DISCUSSION THREAD

**Discussion by Bilge** [symfony/symfony#44991] [Q&A]
What is the correct way to get the name of the matched route or the name of the matched firewall? This information can be found in the Request attributes, but is it just for debugging purposes or is this the intended way to access this information?

![image](https://user-images.githubusercontent.com/470626/149114379-f4cb73e8-b43a-4897-a661-d3e6f14b649c.png)

**Comment by stof** [comment]
for the route name, this is the intended way.
